spec/features/system/admin/instance_management_spec.rb

```diff
@@ -49,7 +49,11 @@
 
       scenario 'I can see all instances' do
         instances = create_list(:instance, 2)
-        visit admin_instances_path(page: Instance.page.total_pages)
+        last_page = Instance.page.total_pages
+        instances &= Instance.order_by_id.page(last_page)
+        expect(instances).not_to be_empty
```

Comment:
@lowjoel This will guarantee at least one instance we created appears in the last page.